.github/workflows/upload-and-deploy.yaml

```diff
@@ -24,45 +24,36 @@ jobs:
     name: Upload files to S3 bucket
     runs-on: ubuntu-latest
     needs: pre-commit
-    if: startsWith(github.ref, 'refs/pull') != true
+    if: github.event_name == 'push' #startsWith(github.ref, 'refs/pull') != true
```

Comment:
I was playing with this. I am playing with the idea of what needs to happen at all on pull request events -- I think everything important is on either push or delete.

Reply:
Agreed - although we can discussed that this means edits to readmes would trigger this step as well.

Reply:
@thomasyu888 I am going to file a JIRA for these workflows to only trigger if the config or src directories are edited